### DISCUSSION TITLE
Uncaught MacroError: The macro you imported from "undefined" is being executed outside the context of compilation with babel-plugin-macros

### DISCUSSION THREAD

**Discussion by AhmadRazaJamal** [ben-rogerson/twin.macro#751] [Q&A]
I just moved my project to a mono repo structure with Vercel's turborepo. I am having a very weird issue where it says the macro is undefined. I tried adding the babel.config file everywhere and the error still exists!

Source of error: one of the packages called shared that uses **twin.macro**. 

The project is written in Next.js and we use tailwind with tw. Any help would be appreciated!

![Screen Shot 2022-11-04 at 5 20 26 PM](https://user-images.githubusercontent.com/33037741/200088330-27cd7958-b0fd-4f2d-829c-987639eb9fb1.png)

**Comment by mosesoak** [comment]
@AhmadRazaJamal Twin isn't yet compatible with Next's SWC compiler. See discussion thread https://github.com/ben-rogerson/twin.macro/discussions/516

You can use Twin with Next by setting `swcMinify: false` in the next config, for now
